Added "--%" to PowerShell command.

jamesche75 opened the pull request:
Based on customer cases, PowerShell has a problem escaping the pipe character. By adding "--%", we can avoid this issue.

azurecla commented:
Hi __@vidarmsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

jamesche75 committed:
Added "--%" to PowerShell command.

Based on customer cases, PowerShell has a problem escaping the pipe character. By adding "--%", we can avoid this issue.
